tests/console-backend-service/internal/domain/serverless/function_test.go

```diff
@@ -0,0 +1,308 @@
+
+	return res.Function, err
+}
+
+func mutationFunction(client *graphql.Client, requestType, arguments, resourceDetailsQuery string) error {
+	query := fmt.Sprintf(`
+		mutation {
+			%s (
+				%s
+			){
+				%s
+			}
+		}
+	`, requestType, arguments, resourceDetailsQuery)
+	req := graphql.NewRequest(query)
+	err := client.Do(req, nil)
+
+	return err
+}
+
+func queryFunctionArguments(nameSuffix, namespace string) string {
+	return fmt.Sprintf(`
+		name: "%s-%s",
+		namespace: "%s"
+	`, FunctionNamePrefix, nameSuffix, namespace)
+}
+
+func deleteManyFunctionsArguments(namespace string, functionsMetadata []FunctionMetadataInput) string {
+	functions := ""
+	for i, meta := range functionsMetadata {
+		functions += fmt.Sprintf(`{name: "%s", namespace: "%s"}`, meta.Name, meta.Namespace)
+		if i+1 < len(functionsMetadata) {
+			functions += ", "
+		}
+	}
+
+	return fmt.Sprintf(`
+		namespace: "%s",
+		functions: [ %s ]
+	`, namespace, functions)
+}
+
+func deleteFunctionArguments(functionSuffix, namespace string) string {
+	return fmt.Sprintf(`
+		namespace: "%s",
+		function: {
+			name: "%s-%s",
+    		namespace: "%s"
+		}
+	`, namespace, FunctionNamePrefix, functionSuffix, namespace)
+}
+
+func mutationFunctionArguments(functionNameSuffix, namespaceName string, labels []string) string {
+	labelTemplate := ""
+	if len(labels) != 0 {
+		for i, label := range labels {
+			labelTemplate += fmt.Sprintf(`%s: "%s"`, label, label)
+			if i+1 < len(labels) {
+				labelTemplate += ", "
+			}
+		}
+	}
+
+	return fmt.Sprintf(`
+		name: "%s-%s",
+		namespace: "%s",
+		params: {
+			labels: { %s },
+			source: "module.exports = { main: function (event, context) { return \"Hello World!\"; } }",
+			dependencies: "{ \"name\": \"asd\", \"version\": \"1.0.0\", \"dependencies\": {} }",
+			env: [  ],
+			replicas: { min: 1, max: 1 },
+			resources: { limits: { memory: "100m", cpu: "128Mi" }, requests: { memory: "50m", cpu: "64Mi" } },
+		},
+	`, FunctionNamePrefix, functionNameSuffix, namespaceName, labelTemplate)
+}
+
+func createFunctionEventArguments(functionSuffix, namespace string) string {
+	return fmt.Sprintf(`
+		namespace: "%s"
+		functionName: "%s-%s"
+	`, namespace, FunctionNamePrefix, functionSuffix)
+}
+
+func functionEventDetailsFields() string {
+	return fmt.Sprintf(`
+        type
+        function {
+			%s
+        }
+    `, functionDetailsFields())
+}
+
+func functionMetadataDetailsFields() string {
+	return `
+		name
+		namespace
+	`
+}
+
+func functionDetailsFields() string {
+	return `
+		name
+		namespace
+		labels
+	`
+}
+
+func subscribeFunctionEvent(client *graphql.Client, arguments, resourceDetailsQuery string) *graphql.Subscription {
+	query := fmt.Sprintf(`
+		subscription {
+			functionEvent (
+				%s
+			){
+				%s
+			}
+		}
+	`, arguments, resourceDetailsQuery)
+	req := graphql.NewRequest(query)
+
+	return client.Subscribe(req)
+}
+
+func fixNamespace(dynamicCli *corev1.CoreV1Client) (*configurer.NamespaceConfigurer, error) {
```

Comment:
Please use https://github.com/kyma-project/kyma/blob/master/tests/console-backend-service/internal/configurer/namespace.go instead of this function

Reply:
Because we already have duplication of helpers in out tests :)

Reply:
This method is only for organizing of namespace creation. This is using `NamespaceConfigurer`